src/index.js

```diff
@@ -1,14 +1,81 @@
 import './style.css';
-import _ from 'lodash';
 
- function component() {
-   const element = document.createElement('div');
+const todoTasks = [
+  {
+    description: 'To do 1',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'to do 2',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'to do 3',
+    completed: true,
+    index: 2,
+  },
+];
 
-  // Lodash, now imported by this script
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-   element.classList.add('hello');
-   
-   return element;
- }
+document.addEventListener('DOMContentLoaded', () => {
+  const todoList = document.getElementById('todo_section');
 
- document.body.appendChild(component());
\ No newline at end of file
+  const form = document.createElement('form');
+  form.classList.add('form');
+
+  const header = document.createElement('div');
+  header.classList.add('header');
+  const heading = document.createElement('span');
+  heading.textContent = 'Today\'s To Do';
+  header.appendChild(heading);
+  const reloadIcon = document.createElement('span');
+  reloadIcon.classList.add('material-symbols-outlined');
+  reloadIcon.textContent = 'cached';
+  header.appendChild(reloadIcon);
+  const input = document.createElement('input');
+  input.classList.add('input');
+  input.placeholder = 'Add to your list...';
+  form.appendChild(header);
+  form.appendChild(input);
+  todoList.appendChild(form);
+
+  const displayTodoList = () => {
+    const unorderedList = document.createElement('ul');
+    unorderedList.classList.add('unorderedList');
+
+    todoTasks.forEach((task) => {
+      const listItem = document.createElement('li');
+      listItem.classList.add('listItem');
+
+      const descriptionSection = document.createElement('div');
+      descriptionSection.classList.add('descriptionSection');
+
+      const checkBox = document.createElement('span');
+      checkBox.classList.add('material-symbols-outlined');
+      checkBox.textContent = 'check_box_outline_blank';
```

Comment:
- [ ] Nice one dynamically rendering the tasks, but kindly note that the `checkbox` or in your case the  `checked` property of the checkbox  should be rendered dynamically also with respect to the `completed` in each objects, meaning it should be checked on `completed: true` and unchecked on completed false...

You could do something like. `<input class='checkbox' type="checkbox" ${task.completed && 'checked'}/>` or more clearer with ternary operator `<input class='checkbox' type="checkbox" ${elem.completed? 'checked' : ''}/>`

the `checked` property is like the `required` property, but what it does is that it triggers the checked icon of the checkbox, when it is present.

in your current case it'll be something like 

```
      const checkBox = document.createElement('span');
      checkBox.classList.add('material-symbols-outlined');
      checkBox.textContent = task.completed ? 'check_box'  : 'check_box_outline_blank';
     
```